mzLib/Readers/ExternalResults/BaseClasses/IQuantifiableRecord.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Readers.ExternalResults.BaseClasses
+{
+    public interface IQuantifiableRecord
+    {
+        /// <summary>
+        /// The full file path to the MS Data file in which the identification was made
+        /// </summary>
+        public string FileName { get; }
+
+        /// <summary>
+        /// A list of tuples, each of which represent a protein. Each tuple contains the accession number, gene name, and organism associated with the given result
+        /// </summary>
+        public List<(string proteinAccessions, string geneName, string organism)> ProteinGroupInfos { get; }
+
+        /// <summary>
+        /// The amino acid sequence of the identified peptide
+        /// </summary>
+        public string BaseSequence { get; }
+
+        /// <summary>
+        /// The amino acid sequence and the associated post-translation modifications of the identified peptide
+        /// </summary>
+        public string ModifiedSequence { get; }
+
```

Comment:
changing the name to FullSequence here creates conflict with the property in MsFraggerPsm with the same name

Reply:
The interface property name is fine. "ModifiedSequence" is used here because that's the name of the corresponding property in the Identification class